ffi-vix_disk_lib.gemspec

```diff
@@ -13,10 +13,9 @@ Gem::Specification.new do |spec|
   spec.homepage      = "http://github.com/ManageIQ/ffi-vix_disk_lib"
   spec.license       = "Apache-2.0"
 
-  spec.files         = `git ls-files -- lib/*`.split("\n")
-  spec.files        += %w(README.md LICENSE.txt)
-  spec.executables   = `git ls-files -- bin/*`.split("\n")
```

Comment:
I don't think we actually wanted to include `bin/setup` in the published gem so I just left this out
```
# ls /opt/manageiq/manageiq-gemset/gems/ffi-vix_disk_lib-1.3.1/bin/
setup
```